solr/core/src/test-files/solr/configsets/cloud-minimal/conf/schema.xml

```diff
@@ -19,10 +19,12 @@
   <fieldType name="string" class="solr.StrField"/>
   <fieldType name="int" class="${solr.tests.IntegerFieldType}" docValues="${solr.tests.numeric.dv}" precisionStep="0" omitNorms="true" positionIncrementGap="0" uninvertible="true"/>
   <fieldType name="long" class="${solr.tests.LongFieldType}" docValues="${solr.tests.numeric.dv}" precisionStep="0" omitNorms="true" positionIncrementGap="0" uninvertible="true"/>
+  <fieldType name="_nest_path_" class="solr.NestPathField"/>
   <dynamicField name="*" type="string" indexed="true" stored="true"/>
   <!-- for versioning -->
   <field name="_version_" type="long" indexed="true" stored="true"/>
   <field name="_root_" type="string" indexed="true" stored="true" multiValued="false" required="false"/>
+  <field name="_nest_path_" type="_nest_path_"/>`
```

Comment:
I was a bit worried about that. Technically I can write the tests without this, using Anonymous children instead, but since that's more of a legacy pattern I figured this was justified.

Also I just grepped for "nest" in the configsets directory and not a single other config has this defined, hopefully that's not a bad omen for test coverage on named child docs 😨.

Reply:
I was speaking out loud too much.  This is fine... it had root already so at this point we just support nested docs better.  Ideally there would be a different schema.